Remotion/Data/DomainObjects/ReflectionUtility.cs

```diff
@@ -37,6 +38,9 @@ public static class ReflectionUtility
     private static readonly Type s_binaryPropertyValueType = typeof (byte[]);
     private static readonly Type s_typePropertyValueType = typeof (Type);
     private static readonly Type s_objectIDPropertyValueType = typeof (ObjectID);
+    private static readonly ConcurrentDictionary<Tuple<Type, Type>, bool> s_canAscribeCache = new();
+    private static readonly ConcurrentDictionary<Tuple<Type, Type>, Type> s_ascribedGenericArgumentsCache = new();
```

Comment:
see earlier remarks on unify caches, no tuple key since it's always ObjectList. And naming should then be specific to the usecase